src/protocols/secure_channel/CASESession.cpp

```diff
@@ -149,7 +149,7 @@ void CASESession::Clear()
     mFabricInfo  = nullptr;
 }
 
-CHIP_ERROR CASESession::Init(SessionManager & sessionManager, SessionEstablishmentDelegate * delegate)
+CHIP_ERROR CASESession::Init(SessionManager & sessionManager, SessionEstablishmentDelegate * delegate, ScopedNodeId peerNodeId)
```

Comment:
I struggled to name this as well. I was trying to avoid leaking out the details of eviction in the API, but I have no better name for it, so I think for now, `sessionEvictionHint` is best.